src/diff/catch-error.js

```diff
@@ -29,8 +30,13 @@ export function _catchError(error, vnode) {
 					component.componentDidCatch(error);
 				}
 
-				if (hasCaught)
-					return enqueueRender((component._pendingError = component));
+				if (hasCaught) {
+					component._pendingError = component;
+					vnode._hydrating = wasHydrating;
```

Comment:
```suggestion
					vnode._hydrating = vnode._hydrating;
```

Reply:
`wasHydrating` is the `vnode._hydrating` value from the original (innermost) vnode, whereas here `vnode` refers to the current value of the loop on L15.